addons/token-exchange/token_exchanger_agent.go

```diff
@@ -134,9 +134,10 @@ func newTokenExchangeAgentController(
 		}
 		return key
 	}
-	informer := spokeSecretInformers.Informer()
+	spokeSecretInformer := spokeSecretInformers.Informer()
```

Comment:
I don't think we need a spoke Secret informer.

Reply:
removed.